.github/workflows/issues.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This example file will enable actions that trigger on created or modified GitHub issues.
+#
+# Note the @maain in `uses:` on the last line. This will call the latest version of the workflow from the `main` branch in the RMI-PACTA/actions repo.
```

Comment:
```suggestion
# Note the @main in `uses:` on the last line. This will call the latest version of the workflow from the `main` branch in the RMI-PACTA/actions repo.
```